io/disk/htx_block_devices.py

```diff
@@ -217,6 +269,7 @@ def stop_htx(self):
             cmd = "htxcmdline -shutdown -mdt %s" % self.mdt_file
             process.system(cmd, timeout=120, ignore_status=True)
 
-        daemon_state = process.system_output('/etc/init.d/htx.d status')
-        if daemon_state.decode("utf-8").split(" ")[-1] == 'running':
-            process.system('/usr/lpp/htx/etc/scripts/htxd_shutdown')
+        #cmd = '/usr/lpp/htx/etc/scripts/htx.d status'
+        #daemon_state = process.system_output(cmd)
+        #if daemon_state.decode("utf-8").split(" ")[-1] == 'running':
+        #    process.system('/usr/lpp/htx/etc/scripts/htxd_shutdown')
```

Comment:
any reason to commentt out all lines ?

Reply:
forgot to remove while pushing.
will be removing this.